packages/core/test/ArrayField_test.js

```diff
@@ -249,6 +273,18 @@ describe("ArrayField", () => {
         .to.be.true;
     });
 
+    it("should add a field when clicking add button even if event is not passed to onAddClick", () => {
+      const { node } = createFormComponent({
+        schema,
+        ArrayFieldTemplate: CustomOnAddClickTemplate,
+      });
+
+      Simulate.click(node.querySelector(".array-item-add button"));
+
+      expect(node.querySelector(".array-item").hasAttribute(ArrayKeyDataAttr))
```

Comment:
Can you instead make the test check just that `.array-item` exists? I don't think we need to check if the key is present, so we should safely be able to remove that check and the data-rjsf-itemkey attribute (which was only a hack needed to check the keys of array items for specific tests regarding array reordering)

Reply:
I fixed the test but I've never used chai... Did i used the right syntax?